### ISSUE TITLE
Playlist wont play Videos

### ISSUE THREAD

**Issue by ckglobalroaming** [jellyfin/jellyfin-android#1480]
### Describe the bug

I have music videos, they are also in playlists. When the playlist is selected on the web client the videos play one after the other.

When the playlist is selected in Jellyfin Android client they are rendered as audio and the CD music interface is displayed. No video is rendered.

Thanks

### Application version

0.17.4

### Where did you install the app from?

Google Play

### Device information

Sony Bravia 4K GB ATV3

### Android version

Android 9 kernel 4.9.125 Bravia_ATV_4KPA-user 9 PTT1.190515.001.S43

### Jellyfin server version

10.9.11

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [X] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by 2bitsin** [comment]
Yes this issue still exists and I have ran into it, I have a playlist of a bunch of videos and it plays as videos on the webclient, but on the android tv app it plays as music, and I found no way to change that!

**Comment by ryanaross** [comment]
This issue still exists on Chromecast with Google TV.

**Comment by X007AceX** [comment]
This issue is still live. 
I have resorted to using Kodi to play video playlists. 
Ive found no plans to fix this issue (or even a breakdown on what the issue is)

So far people have said that if the first item on the playlist is audio it treats the whole playlist as audio. But for mine, I have no audio only files on my server. Thats not the issue.

The playlists work fine on the regular android client, and the web interface.

**Comment by kevgrig** [comment]
I have this issue as well with the latest AndroidTV build and Jellyfin server 10.11.5. What's interesting is some video playlists work, so I copied the first video in a working playlist into the failing video playlist and moved the video to the top and it doesn't play in the failing playlist, so it doesn't appear to be related to the first playlist entry.

On the server, the `playlist.xml` has `<PlaylistMediaType>Video</PlaylistMediaType>`.

**Comment by kevgrig** [comment]
My previous comment was accidentally on the Android project even though my issue was on the AndroidTV project; however, in case this is useful for this issue in the Android project, the issue is the same on AndroidTV and I found the cause is that the PlaybackLauncher checks if any of the items requested to be played are audio and then it decides to play all items through the audio player: https://github.com/jellyfin/jellyfin-androidtv/issues/5276. The PR I submitted is that it checks if the playlist is a video playlist and then plays only video items: https://github.com/jellyfin/jellyfin-androidtv/pull/5277. A quick check in this repository does not show a PlaybackLauncher.kt file, so this may be a different issue.
